build.sbt

```diff
@@ -1,19 +1,37 @@
 import com.typesafe.sbt.packager.docker.{Cmd, ExecCmd}
 
 name := "codacy-bandit"
-
+scalaVersion := "2.13.1"
 version := "1.0.0-SNAPSHOT"
 
-val languageVersion = "2.12.6"
-
-scalaVersion := languageVersion
+lazy val toolVersion = settingKey[String]("The tool version")
+toolVersion := scala.io.Source.fromFile("bandit-version").mkString.trim
+
+lazy val `doc-generator` = project
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.scala-lang.modules" %% "scala-xml" % "1.2.0",
+      "com.github.tkqubo" % "html-to-markdown" % "0.3.0",
+      "org.ccil.cowan.tagsoup" % "tagsoup" % "1.2.1",
+      "com.github.pathikrit" %% "better-files" % "3.8.0",
+      "com.typesafe.play" %% "play-json" % "2.7.4",
+      "com.codacy" %% "codacy-plugins-api" % "3.0.80"
+    ) ++ Seq(
+      "io.circe" %% "circe-core",
+      "io.circe" %% "circe-generic",
+      "io.circe" %% "circe-parser"
+    ).map(_ % "0.12.3"),
```

Comment:
Why play-json and circe? Couldn't we just use one of them?

Reply:
`play-json` was a old, unused import. I removed it.

Reply:
awesome!